core/services/relay/evm/mercury/transmitter.go

```diff
@@ -145,7 +149,14 @@ func (mt *mercuryTransmitter) Start(ctx context.Context) (err error) {
 			return err
 		}
 		mt.wg.Add(1)
-		go mt.runloop()
+
+		transmissions, err := mt.orm.GetTransmitRequests()
```

Comment:
`GetTransmitRequests` should take the start context.

Reply:
I added contexts for the Insert and Delete methods as well.